### PR TITLE
Chat-bot

### PR DESCRIPTION
**Pull request by megha-ghosh109**
# Title 
Chat bot

# Description

Chat bot is created with help of ML neural network tools such as LSTM. The main models used here are sequential, and keras, tensorflow are the main library used here. Model has been trained to 150 epochs to evaluate with very high accuracy.

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have added my Synapse Ticket to the `README.md`
- [x] I have added screenshots  of my project to the `README.md`
- [x] Folder Name = Project Name.

### CODE REVIEW

**Comment by megha-ghosh109** [comment]
added chat bot project

**Review by Priyadarshan2000 — CHANGES_REQUESTED** [review]
@megha-ghosh109 add your Project Title and Project description

**Review by Priyadarshan2000 — APPROVED** [review]
image
Great Job @megha-ghosh109👍 keep contributing 👨🏻‍💻🎉
